DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: igvShiny
 Title: igvShiny: a wrapper of Integrative Genomics Viewer (IGV)
-Version: 0.99.0
+Version: 0.99.1
```

Comment:
Hmm, this version must stay (as required by the Bioc for the initial submission). @karollayna sorry for the confusion.

Reply:
no problem